lib/core/versions/latest/OperationProcessor.ts

```diff
@@ -132,7 +132,8 @@ export default class OperationProcessor implements IOperationProcessor {
         const didUniqueSuffix = anchoredOperationModel.didUniqueSuffix;
         const transactionNumber = anchoredOperationModel.transactionNumber;
         console.debug(
-          `Unable to apply document patch in transaction number ${transactionNumber} for DID ${didUniqueSuffix}: ${SidetreeError.stringify(error)}.`);
+          `Partial update on next commitment hash applied because:
+          Unable to apply delta patches for transaction number ${transactionNumber} for DID ${didUniqueSuffix}: ${SidetreeError.stringify(error)}.`);
```

Comment:
This will print unwanted spaces in log. Consider:
```suggestion
          `Unable to apply delta patches for transaction number ${transactionNumber} for DID ${didUniqueSuffix}: ${SidetreeError.stringify(error)}.`);
```